test-data/unit/fine-grained-dataclass.test

```diff
@@ -0,0 +1,25 @@
+[case replace]
+[file model.py]
```

Comment:
I think this case may be not correct, there should be a top-level section like `import replace`. I am not sure what the test runner does when it is empty. (see `testMetaclassOperators` as a totally random example)

Reply:
It's meaningful to import it? I would think mypy goes over all Python files, rather than follow the import chain.

e.g.
```shell
echo > foo.py
echo "a: str = 42" > bar.py
mypy .
```
results in
```
bar.py:1: error: Incompatible types in assignment (expression has type "int", variable has type "str")  [assignment]
F
```

I don't mind adding the `import` but it doesn't make the test any more/less meaningful (notice that it **does** produce the expected error).

Reply:
I was worried fine-grained test runner doesn't simply check all files (like some other test runners), but I checked the code, and it actually looks OK.